accessibility/index.qmd

```diff
@@ -3,3 +3,95 @@ title: "Accessibility"
 ---
 
 {{< include /includes/_wip.qmd >}}
+
+## Introduction
+
+It is important to us that the outputs of the Seedcase Project
+(including the websites, the software products, and the teaching
+material) are as accessible as possible. This includes people with
+disabilities, people who are older, people who are not native English
+speakers, and people who are not familiar with technology. The Seedcase
+project is no exception. We want to make all output of the Seedcase
+Project as accessible as possible for all potential users. This includes
+both the websites, the software products, and the teaching material.
+
+## Decision drivers
+
```

Comment:
```suggestion
```
This header seems to be a leftover from when this was a decision post. The next section could fit under the Introduction header above.